example/build_with_bazel/brpc_workspace.bzl

```diff
@@ -74,10 +74,15 @@ def brpc_workspace():
         ],
     )
 
-    native.new_local_repository(
-        name = "openssl",
-        path = "/usr",
+    http_archive(
+        name = "openssl",  # 2021-12-14T15:45:01Z
```

Comment:
`example/build_with_bazel/openssl.BUILD` does not build but links to system installed openssl.
Here is the real build:
https://github.com/apache/incubator-brpc/blob/fae30e6a9a84ef8c2e0bfe1ce23afa5611a1ffab/bazel/third_party/openssl/openssl.BUILD#L132